File: Week1/readFile.cpp
#include<iostream>
using namespace std;

int main()
{
    int n;
    cin>>n;
    cout<<"No of test cases "<<n<<endl;
    for(int i=0;i<n;i++)
    {
        int size;
        cin>>size;
        int arr[size];
        for(int i=0;i<n;i++)
        {
            cin>>arr[i];
        }
        int key ;
        cin>>key;
        // linearsearch(arr,size,key);
        cout<<"size "<<size<<endl;
        for(int i=0;i<n;i++)
        {
            cout<<arr[i]<<" ";
        }
        cout<<endl;
        cout<<"key "<<key<<endl;
    }
}

File: Week1/readFile.py
def readFile() :
    n = int(input())
    print("No of test cases ",n)
    for i in range(n) :
        size = int(input())
        array = input()
        array = array.split()
        array = [int(element) for element in array ]
        key = input()
        linearSearch(array,size,key)
        print("size",size)
        print("elements",array)
        print("key",key)

def linearSearch(array,size,key) :
    pass 

if __name__ == '__main__':
    readFile()